runtime-sdk/modules/evm/src/lib.rs

```diff
@@ -559,37 +553,24 @@ impl<Cfg: Config> Module<Cfg> {
         // Run EVM and process the result.
         let (exit_reason, exit_value) = f(&mut executor, gas_limit);
         let gas_used = executor.used_gas();
-        let fee = executor.fee(gas_price);
 
         let exit_value = match process_evm_result(exit_reason, exit_value) {
             Ok(exit_value) => exit_value,
             Err(err) => {
                 <C::Runtime as Runtime>::Core::use_tx_gas(ctx, gas_used)?;
+                Cfg::Accounts::set_refund_unused_tx_fee(ctx, true); // Refund unused fee.
```

Comment:
It would be extra cool to have this be a runtime config item instead of a slightly strange global config, but that would require acknowledging the existence of application-layer failures, so this is probably as good as it gets for now

Reply:
You mean like an EVM config option that would specify whether to refund unused gas on reverts? Would you ever want to set this to false?

Reply:
I'd not set it to false, but I also wouldn't want to have to think about setting it to true in the right places.

Reply:
Ah I see, makes sense.

Reply:
Moved to a config option.